Python/Endpoint Examples/JSON Payload/pdf-with-redacted-text-preview.py

```diff
@@ -0,0 +1,52 @@
+import requests
+import json
+
+with open('/path/to/file', 'rb') as f:
+    upload_data = f.read()
+
+print("Uploading file...")
+upload_response = requests.post(url='https://api.pdfrest.com/upload',
+                    data=upload_data,
+                    headers={'Content-Type': 'application/octet-stream', 'Content-Filename': 'file.pdf', "API-Key": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"})
+
+print("Upload response status code: " + str(upload_response.status_code))
+
+if upload_response.ok:
+    upload_response_json = upload_response.json()
+    print(json.dumps(upload_response_json, indent = 2))
+
+
+    uploaded_id = upload_response_json['files'][0]['id']
+    redaction_options = [{
+        "type": "preset",
+        "value": "uuid",
```

Comment:
```suggestion
        "value": "email",
```

I get this error for this set of options. Replacing with `email` (or some other valid option) worked.

```
{"error": "uuid is not a valid preset redaction option. See documentation for valid options."}
```